### PR TITLE
try to start a task if the pool is not paused when add task/tasks to tasks pool

### PR DESCRIPTION
**Pull request by betterzfz**
try to start a task if the pool is not paused when add task/tasks to tasks pool.

### CODE REVIEW

**Comment by betterzfz** [comment]
try to start a task if the pool is not paused when add task/tasks to tasks pool

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#mac os保存文件夹的自定义属性的隐藏文件
```
Comment (vilic) [inline .gitignore]: Remove this comment.

File: lib/promise-pool.ts [hunk 2]
```diff
@@ -145,8 +145,12 @@ export class Pool<T> {
         this.total += tasksData.length;
         this.pending += tasksData.length;
         this._tasksData = this._tasksData.concat(tasksData);
-
-        this._start();
+        /**
+        * try to start a task if the pool is not paused.
```
Comment (vilic) [inline lib/promise-pool.ts]: Align the stars.

File: lib/promise-pool.ts [hunk 3]
```diff
@@ -145,8 +145,12 @@ export class Pool<T> {
         this.total += tasksData.length;
         this.pending += tasksData.length;
         this._tasksData = this._tasksData.concat(tasksData);
-
-        this._start();
+        /**
```
Comment (vilic) [inline lib/promise-pool.ts]: Add an empty line before the comment.

**Comment by vilicvane** [comment]
thx!

**Pull request merged**